src/main/java/carRacing/Main.java

```diff
@@ -3,14 +3,11 @@
 public class Main {
 
     public static void main(String[] args) {
```

Comment:
```suggestion
    public static void main(String[] args) {
        String carNames = InputView.getCarNames();
        int tryNo = InputView.getTryNo();

        RacingGame racingGame = new RacingGame(carNames, tryNo);
        while(racingGame.isEnd()) {
            ResultView.printResult(racingGame.getCars());
        }
        ResultView.printWinners(racingGame.findWinners()); // RacingResult에서 우승자를 구해 출력
    }
```

이와 같이 RacingGame은 도메인 로직을 수행하는 객체로 바라보고, RacingGame이 실행한 결과 값을 받아 ResultView를 통해 출력하는 방식으로 구현해 보면 어떨까?
도메인 로직과 View로직이 섞이지 않도록 구현함으로써 도메인 로직의 재사용성을 높이는 방식으로 접근해 보면 어떨까?